### ISSUE TITLE
Graphical bug in Aero Fighters 2 + 3

### ISSUE THREAD

**Issue by d4NY0H** [libretro/neocd_libretro#47]
After starting some testing, I have sadly discovered an anoying graphical bug in Aero Fighters (Sonic Wings) 2 + 3. Some of the bigger explosions and the flashing parts of bosses being hit are not dispayed correctly. It looks like the sprites are cut off by some horizontal stripes that are moving from top to bottom. This issue is more noticeable in Aero Fighters 3 but is existing in both titles.

Tested on Raspberry Pi 3, not overclocked, CPU governor set to performance, build neocd_libretro from source, using MAME chd's and REDUMP bin/cue's. I have tried all combinations of different bios files and options without effect on this bug. I have also compared it to latest lr-fbneo cd emulation which is displaying the sprites correctly.

Is there any chance to review this problem?

**Comment by fabrice-martinez** [comment]
I think I see what you mean. Big explosions tend to flicker near the top of the screen. The effect when hitting bosses tend to "flicker" too. Looking closely at the screen I sometimes see a little "line" near the top of the screen because the game is updating its graphics way too late after VBL.

I have checked what happens on real hardware (took some time because I had to grab my Neo Geo CD from the attic) and I can confirm that the same exact thing happens.
NeoCD's behavior in this case is accurate.

I think this is also in part (explosions) linked to the 96 sprites per line limit. Final Burn don't bother emulating that, it's an emulator made with a very different goal in mind:
NeoCD is made for accuracy at the cost of speed.
Final Burn is made for speed, at the cost of accuracy.

The libretro version of NeoCD is derived from an unreleased version called "NeoCDX" that was intended to be used as a simulator to write software for Neo Geo and included a debugger.
I wanted it as accurate as possible to be reasonably sure that if code works on the simulator it works on a real Neo Geo :)
